src/Configurator/CopyFromPackageConfigurator.php

```diff
@@ -33,7 +33,7 @@ public function unconfigure(Recipe $recipe, $config)
         $this->removeFiles($config, $packageDir, getcwd());
     }
 
-    private function copyFiles(array $manifest, string $from, string $to)
+    private function copyFiles(array $manifest, string $from, string $to, array $contextParameters = [])
```

Comment:
You do not use override here as for CopyFromRecipeConfigurator.php ?

Reply:
I should, yes :smile: